api/server/router/container/notify_unsupported.go

```diff
@@ -0,0 +1,11 @@
+//go:build !linux
+// +build !linux
```

Comment:
No need to use the legacy buildtags anymore.

Reply:
Fixed.